File: app/src/main/java/nl/avans/praktijkhoogbegaafd/domain/DayFeeling.java
package nl.avans.praktijkhoogbegaafd.domain;

import java.util.ArrayList;

import nl.avans.praktijkhoogbegaafd.dal.FeelingEntity;

public class DayFeeling {
    private ArrayList<FeelingEntity> feelingsForDay = new ArrayList<>();
    private String date = "";

    public DayFeeling(String date){
        this.date = date;
    }

    public DayFeeling(String date, ArrayList<FeelingEntity> feelings){
        this.feelingsForDay = feelings;
    }

    public ArrayList<FeelingEntity> getFeelingsForDay() {
        return feelingsForDay;
    }

    public void setFeelingsForDay(ArrayList<FeelingEntity> feelingsForDay) {
        this.feelingsForDay = feelingsForDay;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public void addFeeling(FeelingEntity fe){
        feelingsForDay.add(fe);
    }
}

File: app/src/main/java/nl/avans/praktijkhoogbegaafd/domain/DayFeeling.kt
package nl.avans.praktijkhoogbegaafd.domain

import nl.avans.praktijkhoogbegaafd.dal.FeelingEntity
import java.util.ArrayList

class DayFeeling {
    var feelingsForDay = ArrayList<FeelingEntity>()
    var date = ""

    constructor(date: String) {
        this.date = date
    }

    constructor(date: String?, feelings: ArrayList<FeelingEntity>) {
        feelingsForDay = feelings

    }

    fun addFeeling(fe: FeelingEntity) {
        feelingsForDay.add(fe)
    }
}